Automattic/jetpack
# Simple-payments blocks: fix tests

#### Changes proposed in this Pull Request:
* Removes some test fixtures that were committed by mistake, and updates currency on unit tests as the USD currency display differently depending on which env the test is run in

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Check out PR and run `yarn test-extensions` and ensure all tests complete successfully
